gibbon-rts/rts-c/gibbon_rts.c

```diff
@@ -1796,13 +1796,19 @@ void gib_show_usage(char** argv)
     printf("\n");
     printf("This binary was generated by the Gibbon compiler.\n");
     printf("\n");
-    printf("Usage: %s [OPTS] [size] [iters]\n", argv[0]);
+    printf("Usage: %s [OPTIONS...]\n", argv[0]);
 
     printf("\n");
     printf("Options:\n");
-    printf(" --buffer-size <bytes>      Set the buffer size (default %" PRId64 ").\n", gib_global_biginf_init_chunk_size);
-    printf(" --bench-input <path>       Set the input file read for benchmarking. Applies only\n");
-    printf("                            IF the program was *compiled* with --bench-fun. \n");
+    printf(" --biginf-buffer-size <bytes>   Set the buffer size (default %" PRId64 ").\n", gib_global_biginf_init_chunk_size);
+    printf(" --inf-buffer-size <bytes>      Set the buffer size (default %" PRId64 ").\n", gib_global_inf_init_chunk_size);
+    printf(" --bench-input <path>           Set the input file read for benchmarking. Applies only\n");
+    printf("                                IF the program was *compiled* with --bench-fun. \n");
+    printf("\n");
+    printf(" --array-input <path>           Set the file from which to read the array input.\n");
+    printf(" --array-input-length <int>     Set the size of the array input file.\n");
+    printf(" --iterate <int>                Set the number of iterations to perform (default 1).\n");
+    printf(" --size-param <int>             Set the size param (default 1).\n");
```

Comment:
The commentary just recites the name of the parameter, which is not super helpful. Size of what is being specified? The answer could improve the text.

Also, I'd expand `param` to the proper word (`parameter`).

Reply:
sg